### PR TITLE
Update config for resnet50

### PR DESCRIPTION
**Pull request by John1231983**
This is the config for resnet50 that obtain 0.41 LB in my code also use maskRCNN. I did not fully test this PR. I will update the score using this PR

### CODE REVIEW

**Comment by killthekitten** [comment]
Thanks @John1231983!

But I'd like to stick with the default config for now to make it easier for newcomers to spot the difference compared to the original repo.

Could you spend some more of your time to add your config as a separate entity and make it possible to chose the desired config as an argument?

**Comment by John1231983** [comment]
Sure. I will update it when I complete the running. It may be tomorrow. Now, I am running your code with the above setting. Note that, using `USE_MINI_MASK=True` will save memory

**Comment by killthekitten** [comment]
@John1231983 I've merged your branch as is, I think it should be fine for the moment. Please, confirm it works for you.

**Comment by John1231983** [comment]
Yes. It worked with it. I achieve 0.4+ for this configure. Let check in your machine and let me know if it cannot achieve the number. Btw. the function below may the reason make your training code is too slow : 2s/step, while without it takes  1s/step. Do you think the below script is necessary? 
```
        Handle occlusions
        occlusion = np.logical_not(masks[:, :, -1]).astype(np.uint8)
        for i in range(count-2, -1, -1):
            masks[:, :, i] = masks[:, :, i] * occlusion
            occlusion = np.logical_and(occlusion, np.logical_not(masks[:, :, i]))
```

**Comment by killthekitten** [comment]
@John1231983 nice catch, removed that one too. No need for this, I think.

Btw, since we already discuss your config in this PR: how do you perform validation?

P.S. wasn't sure how to properly apply our changes to the original code, so I reverted everything for a couple of times... don't ask :D

**Comment by John1231983** [comment]
For validation, we can separate training to 90 % for training and 10% for validation. However, i did not change it. Just run full training set and submit to Kagge system to see the LB.

**Comment by killthekitten** [comment]
@John1231983 just FYI, it works for me, .302LB after 32 epochs

**Comment by John1231983** [comment]
Great. I hope it achieve 0.4 after 100 epochs. Then reduce learing rate /100 and train 50 epochs you can have 1 to 2 % gain

**Comment by John1231983** [comment]
Hi. Sorry i miss one location for changing. It must be Adam instead of SGD and learning rate is 1e-4. For that, you can achieve 0.4 LB

**Comment by killthekitten** [comment]
@John1231983 don't you mind sending another PR for that? Btw right now it gives .361
